src/Tribe/Views/V2/Template/Event.php

```diff
@@ -55,7 +56,9 @@ public function __construct( Template_Bootstrap $template_bootstrap ) {
 	public function get_path() {
 		$fake_view = View::make( 'reflector' );
 		$path = $fake_view->get_template()->get_template_file( 'index' );
-		return $path;
+		$override = V1_Event_Templates::getTemplateHierarchy( 'default-template' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 2 spaces but found 1 space

<sub>hash: 89dd57ab568d31e9f8340ee4605c0f64</sub>